frontend/src/streamlist/DeleteStreamDialog.jsx

```diff
@@ -22,7 +22,7 @@ export default class DeleteStreamDialog extends React.Component {
 
   confirmDelete() {
     const { deleteId } = this.props;
-    deleteStream(deleteId).finally(this.afterDelete());
+    deleteStream(deleteId).then(this.afterDelete.bind(this));
```

Comment:
the binding should happen in the constructor (at time of creation)

Reply:
Fixed in 61f245fb